test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/SingleDisposalsTriageControllerSpec.scala

```diff
@@ -4081,6 +4103,16 @@ object SingleDisposalsTriageControllerSpec extends Matchers {
         case IndirectDisposal => doc.select("#propertyType-answer").text() shouldBe ""
         case MixedUse         => doc.select("#propertyType-answer").text() shouldBe ""
       }
+    val isSingleDateDisplay: Boolean = completeSingleDisposalTriageAnswers match {
```

Comment:
`isIndirectDisposal` would be better here

Reply:
Done.